bundles/CoreBundle/src/EventListener/Frontend/HardlinkCanonicalListener.php

```diff
@@ -17,12 +17,12 @@
 namespace Pimcore\Bundle\CoreBundle\EventListener\Frontend;
 
 use Pimcore\Bundle\CoreBundle\EventListener\Traits\PimcoreContextAwareTrait;
+use Pimcore\Bundle\StaticRoutesBundle\Model\Staticroute;
```

Comment:
see [ElementListener.php](https://github.com/pimcore/pimcore/pull/13883/files#diff-57746a49d495fb4b2e7c5f5d234b911980876988e503b33427ec5308c8258058)

Reply:
Same here I am not sure how to remove this from the Listener and what would be the implications if I moved this to the bundle.